docs/APDUSPEC.md

```diff
@@ -152,7 +136,7 @@ All other packets/chunks contain data chunks that are described below
 | Field   | Type     | Content              | Expected |
 | ------- | -------- | -------------------- | -------- |
 | Path[0] | byte (4) | Derivation Path Data | 44       |
-| Path[1] | byte (4) | Derivation Path Data | 134      |
+| Path[1] | byte (4) | Derivation Path Data | ?        |
```

Comment:
Path[1] should be 134 and elements 3 and 4 should be removed. This applies for the following commands too